torchci/components/JobSummary.tsx

```diff
@@ -1,11 +1,16 @@
 import { JobData } from "lib/types";
 import JobConclusion from "./JobConclusion";
 
-export default function JobSummary({ job }: { job: JobData }) {
+export default function JobSummary({ job, highlight }: { job: JobData; highlight: boolean; }) {
   return (
     <>
       <JobConclusion conclusion={job.conclusion} />
       <a href={job.htmlUrl}> {job.jobName} </a>
+      {!job.branch? <></> : (highlight ? <b> [{job.branch}] </b> : <span>[{job.branch}]</span>) }
     </>
   );
 }
```

Comment:
All these unary operators are making it hard to read :) 
What about replacing this with something like the following to help with readability:

Some more optimization may be possible to selectively choose the tag type.

```
  var branchName = <></>
  if (job.branch) {
    if (highlight) {
      branchName = (<b> [{job.branch}] </b>)
   } else {
      branchName = (<span>[{job.branch}]</span>)
   }
  }
  return (
    <>
      <JobConclusion conclusion={job.conclusion} />
      <a href={job.htmlUrl}> {job.jobName} </a>
      {branchName}
    </>
  );
}
```